eng/common/templates/job/job.yml

```diff
@@ -2,7 +2,7 @@
 # and some (Microbuild) should only be applied to non-PR cases for internal builds.
 
 parameters:
-# Job schema parameters - https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=vsts&tabs=schema#job
+# Job schema parameters - https://learn.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=vsts&tabs=schema#job
```

Comment:
Undo all changes under eng/common/; these files are overridden every time the Arcade SDK is updated.

Reply:
This comment is not a nit. Please undo the changes in this part of the repo before we merge.